sdcm/cluster.py

```diff
@@ -2968,6 +2999,21 @@ def _node_setup(self, node, seed_address):
         node.remoter.send_files(src=yaml_dst_path,
                                 dst='/tmp/scylla.yaml')
         node.remoter.run('sudo mv /tmp/scylla.yaml /etc/scylla/scylla.yaml')
+
+        rackdc_dst_path = os.path.join(tempfile.mkdtemp(prefix='scylla-longevity'),
+                                       'cassandra-rackdc.properties')
+        node.remoter.receive_files(src='/etc/scylla/cassandra-rackdc.properties',
+                                   dst=rackdc_dst_path)
+        with open(rackdc_dst_path, 'r') as f:
+            rackdc_contents = f.read()
+        rackdc_contents += "\ndc=DataCenter%s\n" % (node.dc_idx + 1)
+        rackdc_contents += "\nrack=RACK1\n"
+        with open(rackdc_dst_path, 'w') as f:
+            f.write(rackdc_contents)
```

Comment:
There is not effect config in cassandra-rackdc.properties before setup, current simple setup can be done by append.

I will update the remote cassandra-rackdc.properties by:
node.remoter.run("sudo -c 'echo \"\ndc=xxx\nrack=RACK1\" >>  /etc/scylla/cassandra-rackdc.properties'")